### PR TITLE
DOCSP-36266 Add Gen AI FAQs

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Adding GenAI FAQ section to the `tools-and-connector` page. 

> [!IMPORTANT]  
> Ignore the Generative AI FAQs side nav element, I think this is a caching issue and should not be present in staging

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/landing/DOCSP-36266/tools-and-connectors/

https://preview-mongodbianfmongodb.gatsbyjs.io/landing/DOCSP-36266/generative-ai-faq/

## JIRA

https://jira.mongodb.org/browse/DOCSP-36266

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65e787b811db7b0427d7fef7

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/tools-and-connectors.txt [hunk 1]
```diff
@@ -64,6 +64,25 @@ Migrators, Tools, and Connectors
...
+Generative AI Information
+-------------------------
+
+.. card-group::
+   :columns: 2
+   :style: extra-compact
+
+   .. card::
+      :headline: Generative AI FAQs
+      :url: https://preview-mongodbianfmongodb.gatsbyjs.io/landing/DOCSP-36266/generative-ai-faq/
```
Comment (ianf-mongodb) [inline source/tools-and-connectors.txt:79]: > [!NOTE]  
> This will be updated to the url above before merge: https://www.mongodb.com/docs/generative-ai-faq/

Comment (ianf-mongodb) [inline source/tools-and-connectors.txt:79]: > [!NOTE]  
> This will be updated to the url above before merge: https://www.mongodb.com/docs/generative-ai-faq/


```suggestion
      :url: https://preview-mongodbianfmongodb.gatsbyjs.io/landing/DOCSP-36266/generative-ai-faq/
```

File: source/generative-ai-faq.txt [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Data processed in Azure OpenAI is not used for any model training. 
+MongoDB has opted out of `asynchronous Abuse Monitoring 
+<https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/abuse-monitoring>`__ 
+so that no prompts or responses are stored with Azure OpenAI. For more detail, 
+refer to the `Azure OpenAI data management policy 
+<https://learn.microsoft.com/en-us/legal/cognitive-services/openai/data-privacy>`__.
+
+What Data Is Being Sent To The Model?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (ltran-mdb2) [inline source/generative-ai-faq.txt:40]: [suggestion] I think we could shorten this for minimalism

```suggestion
What Data Is Sent To The Model?
~~~~~~~~~~~~~~~~~~~~~~~~
```

File: source/generative-ai-faq.txt [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+No.
+
+Does MongoDB Store my Inputs to GenAI Features or the Corresponding Outputs?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB stores inputs and outputs from the GenAI service for 
+a limited time period (up to one year) for troubleshooting and 
+analytics. For any GenAI capabilities offered in our on-premise 
+software (Relational Migrator) you may opt out, in which case 
+MongoDB does not store inputs or outputs from the model.
```
Comment (ltran-mdb2) [inline source/generative-ai-faq.txt:62]: [question] (for my own limited knowledge of GenAI tools) Can you clarify the difference between inputs/outouts of the GenAI service versus inputs/outputs of the model?

File: source/generative-ai-faq.txt [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+==================
+
+FAQ For MongoDB Generative AI Features
+--------------------------------------
+
+What GenAI Features Does MongoDB Currently Offer?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB offers various generative AI (GenAI) capabilities in its products 
+and services, including within Atlas and in its software products.
```
[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/generative-ai-faq.txt:16]: that makes sense! Since "its software products" is covered earlier in the sentence ("in its products and services"), I would suggest taking out software products for minimalism. 

```suggestion
and services, including within Atlas.
```

[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/generative-ai-faq.txt:16]: Or alternatively, 
```suggestion
and services.
```

File: source/generative-ai-faq.txt [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+No.
+
+Does MongoDB Store my Inputs or their Corresponding Outputs to GenAI Features?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB stores inputs and outputs from the GenAI service for 
+up to one year for troubleshooting and 
+analytics. For any GenAI capabilities offered in our on-premise 
+software (Relational Migrator and Compass) you may opt out of input
```
Comment (ltran-mdb2) [inline source/generative-ai-faq.txt:62]: [nit]

```suggestion
software (Relational Migrator and Compass), you may opt out of input 
```

File: source/generative-ai-faq.txt [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+FAQ For MongoDB Generative AI Features
+--------------------------------------
+
+What GenAI Features Does MongoDB Currently Offer?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB offers various Generative AI (GenAI) capabilities in its products 
+and services, including within Atlas and in its software products. 
+For example in MongoDB Compass users can
```
Comment (ltran-mdb2) [inline source/generative-ai-faq.txt:17]: [nit]

```suggestion
For example, in MongoDB Compass users can 
```

File: source/generative-ai-faq.txt [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+What GenAI Features Does MongoDB Currently Offer?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB offers various Generative AI (GenAI) capabilities in its products 
+and services, including within Atlas and in its software products. 
+For example in MongoDB Compass users can 
+leverage natural language to generate MongoDB queries.
+In Charts, Atlas users have the capability to construct 
+visualizations powered by GenAI. Additionally,
```
Comment (ltran-mdb2) [inline source/generative-ai-faq.txt:20]: [suggestion] we could probably make this more active as well

```suggestion
In Charts, Atlas users have the capability to construct 
GenAI powered visualizations. Additionally, 
```

**Review by ltran-mdb2 — APPROVED** [review]
I left a couple small comments but otherwise this LGTM @ianf-mongodb !

**Comment by ianf-mongodb** [comment]
@themantissa This page has been updated per the doc for legal language updates.

File: source/generative-ai-faq.txt [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+For any GenAI capabilities offered in our on-premise software 
+(Relational Migrator and Compass) you may opt out, in which case
+MongoDB will not store your inputs or outputs.
+
+What Agreements Govern My Use of MongoDB's GenAI Features?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Your use of these features is subject to your applicable MongoDB legal 
+agreement, our
+`Acceptable Use Policy <https://www.mongodb.com/legal/terms-of-use>`_ and
```
Comment (themantissa) [inline source/generative-ai-faq.txt:67]: Incorrect link:
```suggestion
`Acceptable Use Policy <https://www.mongodb.com/legal/acceptable-use-policy>`_ and 
```

File: source/generative-ai-faq.txt [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB's current GenAI features are powered by Azure OpenAI. When you 
+use these features, your data is processed in Microsoft Azure using 
+the Azure OpenAI service. 
+
+Data processed in Azure OpenAI is not used 
+for any model training. MongoDB has opted out of asynchronous Abuse 
+Monitoring so that no prompts or responses are stored with Azure OpenAI. 
+For more detail, refer to the Azure OpenAI data management policy.
```
Comment (themantissa) [inline source/generative-ai-faq.txt:35]: We can link if we reword it to:
```suggestion
For more detail, refer to [Azure OpenAI's data management practices](https://learn.microsoft.com/en-us/legal/cognitive-services/openai/data-privacy).
```

File: source/generative-ai-faq.txt [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+What Services Do These GenAI Features Rely On?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB's current GenAI features are powered by Azure OpenAI. When you 
+use these features, your data is processed in Microsoft Azure using 
+the Azure OpenAI service. 
+
+Data processed in Azure OpenAI is not used 
+for any model training. MongoDB has opted out of asynchronous Abuse 
+Monitoring so that no prompts or responses are stored with Azure OpenAI.
```
Comment (themantissa) [inline source/generative-ai-faq.txt:34]:
```suggestion
for any model training. MongoDB has opted out of [asynchronous Abuse 
Monitoring](https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/abuse-monitoring) so that no prompts or responses are stored with Azure OpenAI. 
```

File: source/generative-ai-faq.txt [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+leverage natural language to generate queries or visualizations. And 
+with MongoDB Relational Migrator, GenAI can be leveraged for query 
+conversions.
+
+We also offer a GenAI chatbot within our MongoDB Documentation. 
+
+What Services Do These GenAI Features Rely On?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB's current GenAI features are powered by Azure OpenAI. When you
```
Comment (themantissa) [inline source/generative-ai-faq.txt:28]:
```suggestion
MongoDB's current GenAI features are powered by [Azure OpenAI](https://azure.microsoft.com/en-us/products/ai-services/openai-service). When you 
```

**Review by themantissa — CHANGES_REQUESTED** [review]
Got the okay for the links to remain, with one edit to clarify one.  And one fixed link - it went to ToS not AUP.  TY!

**Review by ianf-mongodb — COMMENTED** [review]
Updated @themantissa!

**Review by themantissa — APPROVED** [review]
Looks great!  Thank you!
